src/components/v5/common/Table/Table.tsx

```diff
@@ -375,18 +386,42 @@ const Table = <T,>({
               {
                 'sm:grid-cols-[1fr_auto_auto]':
                   canGoToNextPage ||
-                  (additionalPaginationButtonsContent && isMobile),
+                  (additionalPaginationButtonsContent &&
```

Comment:
This component already has way too many booleans, but should we wrap all of these into variables? it's getting a bit unreadable